Asap2/Asap2Tree/AXIS_PTS.cs

```diff
@@ -64,7 +64,7 @@ public AXIS_PTS(Location location, string Name, string LongIdentifier, UInt64 Ad
         public CALIBRATION_ACCESS calibration_access;
 
         [Element(12)]
-        public DEPOSIT deposit;
```

Comment:
This do not work if not the corresponding usages is updated.
Line 472 in asap2.language.grammar.y must be updated also.